### ISSUE TITLE
Running bbl plan on MAC OS 14.5 is blocking on invoking bbl-terraform

### ISSUE THREAD

**Issue by mmpkandra** [cloudfoundry/bosh-bootloader#615]
We are observing an issue on invoking the bbl plan on MAC OS 14.5

bbl version : 

```
bbl version
bbl 8.4.103 (darwin/amd64)
```

Terraform version
`Tried with 1.5.7 and lower and also with latest 1.9.1 all giving same error`. `Terraform manager validate version: signal: killed` and following pop up
<img width="253" alt="bbl-terraform" src="https://github.com/cloudfoundry/bosh-bootloader/assets/19312924/6e1fe838-6271-4750-8a5a-a4594d484f93">

**Comment by rkoster** [comment]
Yeah would probably required invoking `spctl --add`. Alternatively, you could run from a container.

**Comment by mmpkandra** [comment]
I found a workaround in cf slack channel: https://cloudfoundry.slack.com/archives/C2DBC3YGZ/p1683058650785429. but it would be nice to have a fix  rather we try the workarounds.

**Comment by beyhan** [comment]
@mmpkandra if you would like to contribute we will be happy to review a pr.

**Comment by ramonskie** [comment]
@mmpkandra i don't use a mac so unfortunately i can not test or add an fix at this moment.
but if you have any spare time to fix it we are happy to review the pr
